### DISCUSSION TITLE
rmd发送的的<setu>会匹配"setu"的"whitegroup"?

### DISCUSSION THREAD

**Discussion by Miuzarte** [Tsuk1ko/CQ-picfinder-robot#215] [一般讨论]
如题, `定时提醒`功能发送的<setu>是否也会受`setu`的`whitegroup`限制

**Comment by Tsuk1ko** [comment]
会，本质是使用你的 `user_id` 在这个群请求 setu

**Reply by Miuzarte** [reply]
非白名单群触发的时候`setuReject`的消息不会发送吗

**Reply by Tsuk1ko** [reply]
应该会发送……

**Reply by Miuzarte** [reply]
> 应该会发送……

我早上在非白名单群创建了一个，刚刚才发现一张都没发
没写错吧应该

ID | 创建者 | crontab | 内容
0 | `user_id` | 30 * * * * | `这是左尖括号`setu`这是右尖括号`

**Reply by Miuzarte** [reply]
> 应该会发送……

在一个白名单群试了下
![IMG_20210615_160741.jpg](https://user-images.githubusercontent.com/66856838/122016709-e79bef00-cd86-11eb-95be-8ba2f7653b95.jpg)

**Reply by Tsuk1ko** [reply]
格式为 `<setu>用于触发机器人色图的文字指令`

这里你应该写成 `<setu>stl`

**Reply by Miuzarte** [reply]
> 格式为 `<setu>用于触发机器人色图的文字指令`
> 
> 这里你应该写成 `<setu>stl`

🌿我还以为指的是关键词

**Reply by Tsuk1ko** [reply]
本质就是定时帮你传这个触发指令给 setu（

同理也可以通过这个来自定义 `r18` 和 `keyword`
